easybuild/easyconfigs/f/fastqsplitter/fastqsplitter-1.2.0-GCCcore-8.2.0-Python-3.7.2.eb

```
@@ -0,0 +1,42 @@
+# This easyconfig was written by the HPC team at Agriculture Victoria Research
+# http://agriculture.vic.gov.au/agriculture/innovation-and-research
+#
+# Author: Ben Moran <basc@agriculture.vic.gov.au>
+easyblock = 'PythonBundle'
+
+name = 'fastqsplitter'
+version = '1.2.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/LUMC/fastqsplitter'
+description = """Splits fastq files evenly."""
+
+toolchain = {'name': 'GCCcore', 'version': '8.2.0'}
+
+builddependencies = [('binutils', '2.31.1')]
+
+dependencies = [('Python', '3.7.2')]
+
+use_pip = True
+
+exts_default_options = {
+    'download_dep_fail': True,
+    'source_urls': [PYPI_SOURCE],
+    'sanity_pip_check': True,
+}
```

@bjmoran with the `PythonBundle` easyblock, download_dep_fail is already True by default, and sanity_pip_check can be used as a normal parameter (below the exts_list):
```suggestion
exts_default_options = {'source_urls': [PYPI_SOURCE]}
```